src/main/java/org/spongepowered/common/mixin/core/world/gen/MixinChunkProviderServer.java

```diff
@@ -116,6 +116,15 @@ public ChunkDataStream getGeneratedChunks() {
         return WorldStorageUtil.doesChunkExist(this.world, this.chunkLoader, chunkCoords);
     }
 
+    // This method exists to allow us to speed up world pre generation.
+    @Override
+    public boolean doesChunkExistSync(Vector3i location) {
```

Comment:
Wait, no, looking at it again, you're right, I don't use it - I'm pointing you in completely the wrong way and I'm not reading it. I'll remove it.